prow/config.yaml

```diff
@@ -1216,9 +1216,6 @@ periodics:
     - name: builtNopImage
       resourceRef:
         name: nop-image
-    - name: builtBashImage
-      resourceRef:
-        name: bash-image
     - name: builtGsutilImage
```

Comment:
The gsutil image was also removed in https://github.com/tektoncd/pipeline/pull/1504 -- sorry for missing both of these!